extensions/amp-a4a/0.1/template-renderer.js

```diff
@@ -15,34 +15,53 @@
  */
 
 import {FriendlyFrameRenderer} from './friendly-frame-renderer';
+import {Renderer} from './amp-ad-type-defs';
 import {getAmpAdTemplateHelper} from './template-validator';
 
 /**
  * Render AMP creative into FriendlyFrame via templatization.
  */
-export class TemplateRenderer extends FriendlyFrameRenderer {
+export class TemplateRenderer extends Renderer {
```

Comment:
this is good. now we also need to change FriendlyFrameRender to a helper util class.

Reply:
Done.